src/main/java/nl/knaw/dans/migration/core/DataverseLoader.java

```diff
@@ -48,11 +50,13 @@ public DataverseLoader(DataverseClient client, ActualFileDAO actualFileDAO, Actu
         this.actualDatasetDAO = actualDatasetDAO;
     }
 
+    @UnitOfWork("hibernate")
```

Comment:
is this a transactional annotation? Does this mean every create is a separate call to the db?